plugins/graphql/package.json

```diff
@@ -21,13 +21,17 @@
   },
   "dependencies": {
     "@backstage/backend-common": "^0.1.1-alpha.17",
+    "@backstage/config": "^0.1.1-alpha.17",
+    "@backstage/plugin-catalog-graphql": "^0.1.1-alpha.17",
```

Comment:
oh so it's essentially a polyglot (is that the word?) package that gets included both by the frontend and the backend? then its package should probably be like `catalog-model` and do

```
  "publishConfig": {
    "access": "public",
    "main": "dist/index.cjs.js",
    "module": "dist/index.esm.js",
    "types": "dist/index.d.ts"
  },
```

and

```
  "scripts": {
    "build": "backstage-cli build",
```

at least that seems to work for that one, but i'm not sure how well we support this situation yet frankly. That one gets away with being very slim and only using very frontend-y packages overall.

Reply:
maybe @Rugvip can chime in later

Reply:
I'm not 100% sure that we'd use this package in the frontend. I think that we would provide a frontend plugin that would provide a wrapped apollo client or something which could generate types against an endpoint? Maybe we could generate those types in the graphql package though or something, not sure yet haven't thought that far in front but I know we want to do something like this